### DISCUSSION TITLE
I don't know if lotus-miner works well

### DISCUSSION THREAD

**Discussion by hsjw12** [filecoin-project/lotus#3779] [General Q&A] [answered]
I installed lotus, but I don't know if lotus-miner works well. Can someone tell me about this.
I got 5FIL from faucet and it goes away too quickly. Is the mining going well?
How can I check if the filecoin installation is successful?

##################################################

Miner: t019***
Sector Size: 32 GiB
Byte Power:   0 B / 189.7 PiB (0.0000%)
Actual Power: 0  / 190 Pi (0.0000%)
        Committed: 0 B
        Proving: 0 B
Below minimum power threshold, no blocks will be won
Deals: 22, 160.1 MiB
        Active: 0, 0 B (Verified: 0, 0 B)

Miner Balance: 0 FIL
        PreCommit:   0 FIL
        Pledge:      0 FIL
        Locked:      0 FIL
        Available:   0 FIL
Worker Balance: 4.520479456209393859 FIL
Market (Escrow):  0 FIL
Market (Locked):  0 FIL

Expected Seal Duration: 12h0m0s

Sectors:
        Total: 4
        PreCommit1: 4

##################################################

Ask: t019***
Price per GiB: 0.0000000005 FIL
Verified Price per GiB: 0.00000000005 FIL
Max Piece size: 32 GiB

###################################################

Daemon:  0.7.0+git.c017d0fb.dirty+api0.14.0
Local: lotus-miner version 0.7.0+git.c017d0fb.dirty

**Comment by ediededi** [comment]
I am in the same situation
did you solve this issue .

**Comment by SgtCoin** [comment, marked as answer]
Archiving for future reference due to age. Network is now v1.13+
